third_party/move/tools/move-smith/src/ast.rs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{names::Identifier, types::Type};
+use arbitrary::Arbitrary;
+use num_bigint::BigUint;
+
+#[derive(Debug, Clone)]
+pub struct CompileUnit {
+    pub modules: Vec<Module>,
+    pub scripts: Vec<Script>,
+}
+
+#[derive(Debug, Clone)]
+pub struct Module {
+    // pub attributes: Vec<Attributes>,
+    // pub address: Option<LeadingNameAccess>,
+    pub name: Identifier,
+    // pub is_spec_module: bool,
+    pub functions: Vec<Function>,
+    pub structs: Vec<StructDefinition>,
+    // pub uses: Vec<UseDecl>,
```

Comment:
I'm ignoring the commented out code for now, but in general, we should not leave commented out code.

Reply:
Will keep this in mind and make sure no commented out code before merging into main